inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/AnnoControllerTest.java

```diff
@@ -118,4 +118,27 @@ void testLogout() throws Exception {
         Assertions.assertFalse(SecurityUtils.getSubject().isAuthenticated());
     }
 
+    @Test
+    void testRegisterFailByInvalidType() throws Exception {
+        UserInfo userInfo = UserInfo.builder()
+                .username("admin11")
+                .password("22222")
```

Comment:
it's better to make the password meaningful. like 'test_username', 'test_password'.
'22222', '88888' is so casual.